docs/data/material/customization/typography/typography.md

```diff
@@ -181,7 +182,7 @@ html {
 }
 ```
 
-_You need to apply the above CSS on the html element of this page to see the below demo rendered correctly_
+You need to apply the above CSS on the HTML element of this page to see the below demo rendered correctly.
```

Comment:
Ah yes, html -> HTML makes sense. https://github.com/mui/material-ui/pull/41516/files#diff-bccc87ba04be49b29a35bdea3180dcca13d10b8f674a0cb9342cb36277b2e65eR14 should cover them all.